source/libs/page-detect.ts

```diff
@@ -0,0 +1,92 @@
+/* eslint-disable unicorn/prefer-starts-ends-with */
+/* The tested var might not be a string */
+
+import {check as isReserved} from 'github-reserved-names';
+import {getUsername, getCleanPathname, getRepoPath, getOwnerAndRepo} from './utils';
+
+export const is404 = (): boolean => document.title === 'Page not found · GitHub';
```

Comment:
Oops, I forgot you didn't want explicit returns. I'll fix that tomorrow or Sunday.

Reply:
I think @sindresorhus wanted to enable that at some point, so it’s ok